config/environments/test.rb

```diff
@@ -41,6 +41,10 @@
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
 
+  # Unlike controllers, the mailer instance doesn't have any context about the
+  # incoming request so you'll need to provide the :host parameter yourself.
+  config.action_mailer.default_url_options = { host: 'www.example.com' }
```

Comment:
I think this is safe given test env, but could use more review ... we have the `1_hosts` intializer doing some variation on this.

Sort of related - various ongoing discussion about aligning the diff portions of the spec suite to each other re: hostnames/ports/etc.